File: frontend/fix.py
import os

files = [
    r'src\pages\PatientDashboard.tsx',
    r'src\pages\DoctorDashboard.tsx',
    r'src\pages\AppointmentsPage.tsx',
    r'src\pages\BookingWizard.tsx'
]

for f in files:
    if os.path.exists(f):
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
        
        # Replace escaped backticks
        content = content.replace(r'\`', '`')
        # Replace escaped dollar signs inside template literals
        content = content.replace(r'\${', '${')
        
        with open(f, 'w', encoding='utf-8') as file:
            file.write(content)
        print(f'Fixed {f}')

File: frontend/fix.js
const fs = require('fs');
const path = require('path');

const files = [
  'src/pages/PatientDashboard.tsx',
  'src/pages/DoctorDashboard.tsx',
  'src/pages/AppointmentsPage.tsx',
  'src/pages/BookingWizard.tsx'
];

files.forEach(f => {
  const p = path.join(__dirname, f);
  if (fs.existsSync(p)) {
    let content = fs.readFileSync(p, 'utf8');
    content = content.replace(/\\`/g, '`');
    content = content.replace(/\\\${/g, '${');
    fs.writeFileSync(p, content, 'utf8');
    console.log('Fixed', f);
  }
});
